opentech/apply/funds/workflow.py

```diff
@@ -137,6 +137,8 @@ def can_staff_edit(self, user: 'User') -> bool:
 
 Request = Stage('Request', False)
 
+RequestExt = Stage('RequestExt', True)
```

Comment:
Ideas for a better name then "RequestExt"?

Reply:
Does this need to be different? We display it on the submission detail page as part of the meta information. The fact it has an extra stage for the External Review is covered by the status bar?

![image](https://user-images.githubusercontent.com/6338661/47366821-5c9df000-d6d6-11e8-896c-96a5909a968f.png)

Thus:
```suggestion
RequestExt = Stage('Request', True)
```